Scarb.toml

```diff
@@ -10,8 +10,8 @@ license-file = "LICENSE"
 keywords = ["access control", "authorization", "cairo", "starknet"]
 
 [dependencies]
-starknet = "2.3.1"
-snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.11.0" }
+starknet = "2.4.0"
+snforge_std = { git = "https://github.com/foundry-rs/starknet-foundry.git", tag = "v0.13.1" }
```

Comment:
Moving `snforge_std` to `[dev-dependencies]` still causes the test suite to fail compilation.

Reply:
Ok, weird. Let's keep it then, we can ask the Foundry team afterwards why is that.